### DISCUSSION TITLE
1️⃣2️⃣3️⃣ aren't rendered properly

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#128125] [Mobile] [answered]
GitHub Mobile beta for Android 

- https://github.com/community/community/discussions/48914#discussioncomment-9373567

![Screenshot_20240611-074217.png](https://github.com/community/community/assets/2119212/efa41abc-f371-4cb6-9891-886765709d1f)

![Screenshot_20240611-074408.png](https://github.com/community/community/assets/2119212/0c4f09b1-5cb7-481d-9562-6616740113f0)

1️⃣ Close as Out of Date: If the topic is no longer relevant, close the Discussion as out of date at the bottom of the page.

2️⃣ Provide More Information: Share additional details or context — or let the community know if you've found a solution on your own.

3️⃣ Mark a Reply as Answer: If your question has been answered by a reply, mark the most helpful reply as the solution.

**Comment by elisealix22** [comment, marked as answer]
Thanks @jsoref ! I can reproduce this as well. We'll look into it.

**Comment by ghost** [comment]
Fix
